src/components/flowchart/flowchart.test.js

```diff
@@ -189,6 +189,22 @@ describe('FlowChart', () => {
       type: 'UPDATE_ZOOM'
     });
   });
+
+  it('will not perform zoom operations for Infinity zoom values', () => {
+    const dispatch = jest.fn();
+
+    const zoom = { scale: 1, x: Infinity, y: Infinity };
+    mapDispatchToProps(dispatch).onUpdateZoom(zoom);
+    expect(dispatch.mock.calls.length).toEqual(1);
+  });
+
+  it('will not perform zoom operations for NaN zoom values', () => {
+    const dispatch = jest.fn();
+
+    const zoom = { scale: 1, x: NaN, y: NaN };
+    mapDispatchToProps(dispatch).onUpdateZoom(zoom);
+    expect(dispatch.mock.calls.length).toEqual(1);
+  });
```

Comment:
These tests don't fail when you change the value of x/y to numbers, e.g.
```es6
const zoom = { scale: 1, x: 1, y: 1 };
```

If the checks were being applied here, I'd expect `dispatch.mock.calls.length` to equal 0 when x/y are `NaN`. These tests need to be completely reworked.

Reply:
great spot Richard, thank you for your check - I'll rework this test and test them with the other conditions. Thanks again!

Reply:
on having further investigated into setting up tests to test the zoom behaviour for flowchart, we have concluded that it would some further effort in look into the test setup, where this is best to be seperated in a seperate ticket whilst unblocking the merge of this fix to solve the problems of the reported errors in the console. 

Hence I have deleted those tests for now and they will be added in via https://jira.quantumblack.com/browse/KED-2363